content/components/tooltip.mdx

```diff
@@ -9,10 +9,10 @@ cssId: tooltip
 ---
 
 import ComponentLayout from '~/src/layouts/component-layout'
-import {Link, Text} from '@primer/react'
+import {Box, Heading, Link, Text, Note} from '@primer/react'
```

Comment:
I think that `Box`, `Heading` and `Link` aren't use in this file 🤔

Reply:
whoops. leftover artifact from messing with the tooltip file before making the a11y link component. good catch!